.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
+ For ci, I will implement it in our project. Could you recommend some learning resources? :)

Reply:
I did it first time :P copy paste guy, uncle google and testing